### DISCUSSION TITLE
Can I increase the page width?

### DISCUSSION THREAD

**Discussion by iouzzrj** [edrlab/thorium-reader#2061] [Q&A] [answered]
I use scrollable mode to read epubs. As you can see, there's huge amount of whitespaces besides the text of the ebook. Can I increase the page width?
![屏幕截图 2024-01-24 160428](https://github.com/edrlab/thorium-reader/assets/43952332/85dc0ce6-08ab-40af-abe0-c82ea3696476)

**Comment by iouzzrj** [comment]
I saw there are some discussions about page width here: https://github.com/edrlab/thorium-reader/discussions/1777.
So I can modify the source of reader to achieve this goal?

**Reply by danielweck** [reply, marked as answer]
Hello, this behaviour in Thorium is dictated by an opinionated styling/formatting library called ReadiumCSS, which limits the "line length" in both scroll and paginated modes (thus affecting the perceived "margins" on either side  of the content).

In a nutshell, here is the CSS injected into the `body` element:

`--RS__maxLineLength: 40rem; max-width: var(--RS__maxLineLength);`

I filed an issue in the ReadiumCSS GitHub repository: https://github.com/readium/readium-css/issues/130

Thank you for your feedback!

**Comment by tor-soft** [comment]
Interested in this as well, especially for single column if possible
